.prettierrc

```diff
@@ -10,7 +10,5 @@
   "requirePragma": false,
   "semi": false,
   "singleQuote": true,
-  "tabWidth": 2,
-  "trailingComma": "all",
-  "useTabs": false
```

Comment:
This corresponds to the addition of the EditorConfig-managed [`indent_style`](https://github.com/codecov/uploader/blob/0743c8b/.editorconfig#L9).

Reply:
🤦‍♀️ I thought that was this file. Too early 😹